modules/nw-enabling-multicast.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * networking/using-multicast.adoc
+
+[id="enabling-multicast-{context}"]
+= Enabling multicast between pods
+
+You can enable multicast between pods for your project.
+
+.Prerequisites
+
+* The `oc` CLI program installed.
```

Comment:
I would reword this to say "Install the `oc` CLI program." Though instead of "program", I would either say "tool" or just leave it off and just say "`oc` CLI".

Reply:
I'm curious about the best way to handle this, so I added an item to the agenda.